corehq/apps/userreports/tests/test_columns.py

```diff
@@ -134,7 +141,9 @@ def _new_case(self, properties):
             update=properties,
         ).as_xml()
         post_case_blocks([case_block], {'domain': self.domain})
-        return CommCareCase.get(id)
+        case = CommCareCase.get(id)
```

Comment:
Yeah. that's probably true. although I don't think it's necessary as it was already here and that's still the default